artifactory/utils/oc/getimagedetails.go

```diff
@@ -0,0 +1,34 @@
+package oc
+
+import (
+	"fmt"
+	"github.com/pkg/errors"
+	"strings"
+
+	gofrogcmd "github.com/jfrog/gofrog/io"
+	"github.com/jfrog/jfrog-client-go/utils/errorutils"
+)
+
+func GetImageDetails(executablePath, ocBuildName string) (imageTag, manifestSha256 string, err error) {
+	getImageDetailsCmdConfig := createGetImageDetailsCmdConfig(executablePath, ocBuildName)
+	output, err := gofrogcmd.RunCmdOutput(getImageDetailsCmdConfig)
```

Comment:
I recommend not to use gofrogcmd.RunCmdOutput whenever possible, due to the overhead it brings, but use go's command execution API directly. See how we executed jfrog-cli plugins as a reference.
The same goes for the code in oc/version.go.